src/main/kotlin/com/hivemq/extension/gradle/HivemqExtensionExtensionImpl.kt

```diff
@@ -35,6 +35,7 @@ abstract class HivemqExtensionExtensionImpl @Inject constructor(
     final override val author = objectFactory.property<String>()
     final override val priority: Property<Int> = objectFactory.property<Int>().convention(0)
     final override val startPriority: Property<Int> = objectFactory.property<Int>().convention(1000)
+    final override val startMandatory: Property<Boolean> = objectFactory.property<Boolean>().convention(null)
```

Comment:
```suggestion
    final override val startMandatory: Property<Boolean> = objectFactory.property<Boolean>()
```
not set is the default

Reply:
No, don't remove it from all lines, some calls have undefined nullability (generics coming from Java APIs).
For this line, we are calling a Kotlin API where the generic also has proper nullability, so the explicit type is unnecessary.